MAUI/TreeView/empty-view.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: EmptyView in .NET MAUI TreeView Control | Syncfusion
+description: Learn here about EmptyView support in Syncfusion .NET MAUI TreeView (SfTreeView) Control and more about it.
+platform: MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Empty view in .NET MAUI TreeView (SfTreeView)
+
+The `SfTreeView` control allows you to display and customize the empty view content when no data is available. The `EmptyView` property can be set to a string or a view, which will be displayed when the `ItemsSource` is null or when the collection bound to the `ItemsSource` is empty and `EmptyViewTemplate` is used to customize the appearance of `EmptyView`.
+
+## Display a string when treeView has no items
+
+The `EmptyView` property can be set to a string.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.TreeView;assembly=Syncfusion.Maui.TreeView">
+  <syncfusion:SfTreeView x:Name="treeView"
+                         ItemsSource="{Binding Items}"
+                         EmptyView="No Items">
+  </syncfusion:SfTreeView>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+treeView.EmptyView = "No Items";
```

Comment:
I already mentioned in previous review that c# codes should be equivalent to the xaml. why it was missed?